srcpkgs/volta/patches/openssl-sys-fix-version.patch

```diff
@@ -0,0 +1,11 @@
+--- Cargo.toml
++++ Cargo.toml
+@@ -39,6 +39,7 @@ structopt = "0.2.14"
+ rand = "0.5"
+ cfg-if = "0.1"
+ mockito = { version = "0.14.0", optional = true }
++openssl-sys = { version = "0.9.53" }
```

Comment:
Why?

Reply:
My guess, libressl compatibility issue.
@atk you can just add `cargo update --package openssl-sys --precise 0.9.53` to e.g. `pre_build()` in the template instead of writing a patch.

Reply:
Because without pinning the version, it will not compile with libressl-3, which is the reason the currently available binary is not working with void linux.

Reply:
Since the package is only added implicitly to other dependencies, it needs to be `cargo add`, but yes, I get the idea.

Reply:
fixed.